qlty-cli/src/commands/auth/login.rs

```diff
@@ -1,13 +1,41 @@
+use std::io::stdin;
+
 use crate::{Arguments, CommandError, CommandSuccess};
 use anyhow::Result;
 use clap::Args;
-use qlty_cloud::load_or_retrieve_auth_token;
+use console::style;
+use qlty_cloud::{load_or_retrieve_auth_token, store_auth_token};
 
 #[derive(Args, Debug)]
-pub struct Login {}
+pub struct Login {
+    /// Provide a CLI token manually or use "-" to read from standard input (see https://qlty.sh/user/settings/cli)
```

Comment:
This help string may be a bit long and cause wrapping. I generally try to keep these to a length of ~60-70 max

Reply:
The wrapping is handled by clap:

```
Options:
      --token <TOKEN>     Provide a CLI token manually or use "-" to read from standard input (see
                          https://qlty.sh/user/settings/cli)
```

Reply:
Ah lovely, didn't realize that. Thanks